stetho/src/main/java/com/facebook/stetho/inspector/elements/Descriptor.java

```diff
@@ -55,6 +58,44 @@ public final void postAndWait(Runnable r) {
     getHost().postAndWait(r);
   }
 
+  public void setAttributesAsText(Object element, String text) {
```

Comment:
I think I see what you're doing here -- you want the default implementation to be a no-op, right? Let's not put this in `Descriptor`, and leave that up to `ChainedDescriptor` (none of the other methods on `NodeDescriptor` have any implementation in `Descriptor`). Then in `ObjectDescriptor`, which is the only one that doesn't derive from `ChainedDescriptor`, we'll leave it as an empty no-op method, which is what `ChainedDescriptor` will eventually call into anyway if it's not overriden.